persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java

```diff
@@ -57,6 +63,16 @@ private PersistToolsConstants() {
     public static final String PORT = "port";
 
     public static final String CREATE_DATABASE_SQL = "CREATE DATABASE %s";
+    public static final String KEYWORD_PROVIDER = "provider";
+    public static final String KEYWORD_SHADOWDATABASE = "shadowDatabase";
+    public static final String KEYWORD_PERSIST = "persist";
+    public static final String KEYWORD_MODULES = "modules";
```

Comment:
Isnt it more appropriate to rename these are sub module names _x?

Reply:
updated in [8415ac4](https://github.com/ballerina-platform/persist-tools/pull/52/commits/8415ac40b0d7895fed58315bbc7624580d898835)